api/v1alpha1/hosted_controlplane.go

```diff
@@ -25,9 +25,14 @@ type HostedControlPlane struct {
 
 // HostedControlPlaneSpec defines the desired state of HostedControlPlane
 type HostedControlPlaneSpec struct {
+	// ReleaseImage is the release image applied to the hosted control plane.
 	ReleaseImage string                      `json:"releaseImage"`
 	PullSecret   corev1.LocalObjectReference `json:"pullSecret"`
-	IssuerURL    string                      `json:"issuerURL"`
+	// IssuerURL is an OIDC issuer URL which is used as the issuer in all
+	// used by the service account token issuer. The secret is expected to point to
+	// an S3 bucket which contains the OIDC authentication inside
+	// (E.G https://hypershift-ci-1-oidc.s3.us-east-1.amazonaws.com/sample-dev-6r5rf)
+	IssuerURL string `json:"issuerURL"`
```

Comment:
Is this field provider specific, i.e., do we always expect to point to s3?

Reply:
Ohh that was my bad, it's not provider specific. Let me fix it :)